workspaces/analytics/plugins/analytics-module-matomo/src/index.ts

```diff
@@ -21,6 +21,7 @@ import {
 } from '@backstage/core-plugin-api';
 import { MatomoAnalytics } from './api';
 
+export { matomoModule as default } from './module';
```

Comment:
This is adding the New Frontend System support form the default export and in all other cases where we've added this it's been from the `/alpha`. Is this for sure how you want to move forward?